Add missing include of cstdint when building with GCC 13.
Fix error: ‘uintptr_t’ was not declared in this scope
Author: Emik Lin (HKUMed CPOS)
--- src/cxx/include/common/Logger.hh.orig	2017-06-23 01:14:50.000000000 +0800
+++ src/cxx/include/common/Logger.hh	2025-09-16 15:40:07.692965529 +0800
@@ -17,6 +17,7 @@
 
 #include <ios>
 #include <string>
+#include <cstdint>
 
 #include <boost/atomic.hpp>
 #include <boost/lexical_cast.hpp>
